### ISSUE TITLE
Can't open pdb file ...\packages\sqlite-net-pcl.1.4.118\lib\netstandard1.1\SQLite-net.pdb

### ISSUE THREAD

**Issue by aphex3k** [praeclarum/sqlite-net#614]
I'm trying to build a Windows Phone 8 project from command line using MSBuild. Using Visual Studio 2015 everything seems to be working ok, but from command line I get the following error:

    _GenerateAppxSymbolPackage:
    Can't open pdb file <redacted>.mobile\packages\sqlite-net-pcl.1.4.118\lib\netstandard1.1\SQLite-net.pdb

I fails creating the debug symbols and subsequently errors out at the `CreateAppStoreContainer` step due to the missing .appxsym.

This is the command line i use to kick of the build:

    "C:\Program Files (x86)\MSBuild\14.0\bin\amd64\MSBuild.exe" /target:<redacted>_WindowsPhone8 /p:Configuration=Release "/p:Platform=Any CPU" /m /p:DebugSymbols=false /p:DebugType=None <redacted>.mobile.sln

Please let me know if what more information you need or maybe I'm doing it wrong?

FWIW the Windows 10 UWP build works fine from command line...

**Comment by aphex3k** [comment]
I think when using Visual Studio to add the nuget package it adds the _wrong_ reference. It adds `lib\netstandard1.1` when it should add `lib\portable-net45+wp8+wpa81+win8+MonoAndroid10+MonoTouch10+Xamarin.iOS10`?!

**Comment by praeclarum** [comment]
Yeah darn. The PDB file is the new portable format - perhaps VS 2015 can't understand it. I would have assumed it would just ignore it then. I guess I could remove it?

**Comment by aphex3k** [comment]
other pdbs seem to work ok though? Im not to familiar with msbuild so I can only guess but I have a hunch your question was rethorical?

**Comment by praeclarum** [comment]
Yeah the format changed but the extension remained the same. Unfortunately I use mono to build everything and can't generate the old format. I'll strip them out.

**Comment by aphex3k** [comment]
Thanks Frank!

**Comment by schuettecarsten** [comment]
Please reopen this issue because the problem is still there - as a workaround, set debugtype to "pdbonly" in csproj file when you create the nuget package.

**Comment by SBortz** [comment]
Same problem here.
